databook/physiology/ephys/np-ultra/npultra-psychedelics-stimuli.md

```diff
@@ -0,0 +1,14 @@
+# NP Ultra & Psychedelics Stimuli and Optotagging protocol
+
+## Spontaneous Activity
+Each session includes at least three epochs of spontaneous activity. During this epoch the monitor is held at mean luminance gray and there is no patterned stimulus presented. This provides a valuable time that can be used as a baseline comparison for visually evoked activity.
```

Comment:
how long are the spontaneous epochs? If it's something consistent, it'd be useful to say how long.